doc/active_docs/Account Management API.json

```diff
@@ -6509,7 +6509,7 @@
       ]
     },
     {
-      "path": "/admin/api/services/{service_id}/backend_apis.json",
+      "path": "/admin/api/services/{service_id}/backend_usage.json",
```

Comment:
should be plural I think

Reply:
and plural would be `backends_usages` in this case i think.

(The noun usage can be countable or uncountable.

In more general, commonly used, contexts, the plural form will also be usage.

However, in more specific contexts, the plural form can also be usages e.g. in reference to various types of usages or a collection of usages.)

Reply:
u r right but it is done in the next commit :+1:

Reply:
I'm not sure about this.
if it's plural because it's multiple backends, then it's backend_usage. If it's plural because it's multiple types of usage then it could be usages.
1 dog being used - dog_usage
2 dogs being used - dog_usage
ways to use dogs, for example as a pet or as food, then dog_usages is OK.
i have no idea if you can connect this to your code but maybe it makes you think about your dog. ;)

Reply:
it's plural because it's multiple usages of multiple backends, each backend only used once, so not sure @secretsurf

Reply:
OK, for the English language then, the safest is "backend_usage" because it's OK for all cases. Usages is (barely) OK only for one case and even then would cause a raised eyebrow or two. But if there needs to be a method name that is singular and another that's plural then usages it is.